uint8 force

uint8 position

uint8 speed
